TheBereftSouls.cs

```diff
@@ -6,13 +6,34 @@ public class TheBereftSouls : Mod
 {
     internal static TheBereftSouls Instance = null!;
 
+    public static Mod calamity = null!;
```

Comment:
make all of these `{get; private set;}`

Reply:
MB, for this one 
https://github.com/bromeex/TheBereftSouls/pull/54#discussion_r1875195332